mirrors.d/mirrors.webhs.pt.yml

```
@@ -0,0 +1,12 @@
+name: mirrors.webhs.pt
+address:
+  http: http://mirrors.webhs.pt/almalinux/
+  https: https://mirrors.webhs.pt/almalinux/
+  rsync: rsync://mirrors.webhs.pt/almalinux/
+update_frequency: 3h
+sponsor: WebHS
+sponsor_url: https://mirrors.webhs.pt/
+email: noc@webhs.pt
+geolocation:
+  country: PT
+  city: Lisbon
```

```suggestion
  state_province:
  city: Lisbon
```

`state_province` is required even if blank.